src/main/java/co/rsk/bitcoinj/script/StandardRedeemScriptParser.java

```diff
@@ -25,14 +25,12 @@ public class StandardRedeemScriptParser implements RedeemScriptParser {
 
     @Override
     public int getM() {
-        checkArgument(redeemScriptChunks.get(0).isOpCode());
-        return Script.decodeFromOpN(redeemScriptChunks.get(0).opcode);
+        return RedeemScriptValidator.decodeN(redeemScriptChunks.get(0));
     }
 
     @Override
     public int findKeyInRedeem(BtcECKey key) {
-        checkArgument(redeemScriptChunks.get(0).isOpCode()); // P2SH scriptSig
-        int numKeys = Script.decodeFromOpN(redeemScriptChunks.get(redeemScriptChunks.size() - 2).opcode);
+        int numKeys = RedeemScriptValidator.decodeN(redeemScriptChunks.get(redeemScriptChunks.size() - 2));
```

Comment:
Probably better to split this in 2 lines to understand what redeemScriptChunks.size() - 2 means